### PR TITLE
Performance Tab - Display sheriffed results on top, unsheriffed on the bottom

### PR DESCRIPTION
**Pull request by beatrice-acasandrei**

### CODE REVIEW

File: ui/job-view/details/tabs/PerformanceTab.jsx
Comment (beatrice-acasandrei) [inline ui/job-view/details/tabs/PerformanceTab.jsx:83]: Removed sorting in alphabetical order.

**Review by esanuandra — APPROVED** [review]

**Comment by davehunt** [comment]
Hey @beatrice-acasandrei, do you have a screenshot to show how this looks?

**Comment by beatrice-acasandrei** [comment]
> Hey @beatrice-acasandrei, do you have a screenshot to show how this looks?

@davehunt Sure, this is how it looks with the staging database, I've ordered the results in this order by `should_alert`: first true values, then null values and at the end false values.
perf-tab

Let me know if there's anything I should change.

**Comment by davehunt** [comment]
In this case it is just `fetch_content` that doesn't alert? I'm fine with this as it is, and don't want to hold up landing it, but I do wonder if we should make it clear that some results don't alert. I'm thinking we could have two separate tables, or perhaps hide results that don't alert by default until a user expands them. What do you think? Possibly a good conversation for the next projects meeting.

**Comment by beatrice-acasandrei** [comment]
> In this case it is just `fetch_content` that doesn't alert? I'm fine with this as it is, and don't want to hold up landing it, but I do wonder if we should make it clear that some results don't alert. I'm thinking we could have two separate tables, or perhaps hide results that don't alert by default until a user expands them. What do you think? Possibly a good conversation for the next projects meeting.

Yes, only `fetch_content` doesn't alert. I agree with both suggestions, the second one seems an extension of the first one to me, since we would still display a table for the ones that don't alert. I will add this to the agenda of the next projects meeting so we can discuss it.

**Review by esanuandra — APPROVED** [review]

**Comment by beatrice-acasandrei** [comment]
@jmaher Could you take a look on this when you have time? Seems that @esanuandra doesn't have enough rights (does not have write access) and I can't merge PRs with only her approval.

**Review by alexandru-io — APPROVED** [review]

**Pull request merged**
